### ISSUE TITLE
IsADirectoryError with conda v4.4.10 

### ISSUE THREAD

**Issue by brospars** [jupyterhub/jupyterhub#1950]
Hi,

I'm working with the lastest jupyterhub version and it seems that the [updated conda version](https://github.com/jupyterhub/jupyterhub/commit/f99b7cb7eb7ee12657fe89163ddd634e2898b5eb#diff-3254677a7917c6c01f55212f86c57fbf) is bugged (cf : https://github.com/conda/conda/issues/6811)

**How to reproduce the issue**
Here's my dockerfile
```docker
ARG JUPYTERHUB_VERSION
FROM jupyterhub/jupyterhub-onbuild:0.9.0.dev

# Install dockerspawner, oauth, postgres
RUN /opt/conda/bin/conda install -yq psycopg2=2.7
RUN /opt/conda/bin/conda clean -tipsy
RUN /opt/conda/bin/pip install --no-cache-dir dockerspawner==0.9.*
```

**What you expected to happen**
Successful build, it's working if I downgrade conda to 4.2.12 as before

**What actually happens**
```sh
Sending build context to Docker daemon   2.56kB
Step 1/5 : ARG JUPYTERHUB_VERSION
Step 2/5 : FROM jupyterhub/jupyterhub-onbuild:0.9.0.dev
# Executing 1 build trigger
 ---> 5f595f0e5586
Step 3/5 : RUN /opt/conda/bin/conda install -yq psycopg2=2.7
 ---> Running in 8a2e13c5261e
Solving environment: ...working... done

## Package Plan ##

  environment location: /opt/conda

  added / updated specs: 
    - psycopg2=2.7


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    psycopg2-2.7.4             |   py36hb7f436b_0         292 KB
    ca-certificates-2018.03.07 |                0         124 KB
    libpq-9.6.6                |       h4e02ad2_0         101 KB
    openssl-1.0.2o             |       h20670df_0         3.4 MB
    certifi-2018.4.16          |           py36_0         142 KB
    ------------------------------------------------------------
                                           Total:         4.1 MB

The following NEW packages will be INSTALLED:

    libpq:           9.6.6-h4e02ad2_0                
    psycopg2:        2.7.4-py36hb7f436b_0            

The following packages will be UPDATED:

    certifi:         2018.4.16-py36_0     conda-forge --> 2018.4.16-py36_0 
    openssl:         1.0.2o-0             conda-forge --> 1.0.2o-h20670df_0

The following packages will be DOWNGRADED:

    ca-certificates: 2018.4.16-0          conda-forge --> 2018.03.07-0     



==> WARNING: A newer version of conda exists. <==
  current version: 4.4.10
  latest version: 4.5.4

Please update conda by running

    $ conda update -n base conda



IsADirectoryError(21, 'Is a directory')


The command '/bin/sh -c /opt/conda/bin/conda install -yq psycopg2=2.7' returned a non-zero code: 1

```

**Comment by willingc** [comment]
Hi @brospars,

I've opened a PR for the Dockerfile in the root of the jupyterhub repo to bump the version of miniconda used.

**Comment by brospars** [comment]
@willingc Perfect thank you, I should have done that directly

**Comment by willingc** [comment]
Closed by #1952
